data-prepper-plugins/http-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/http/AwsRequestSigningApacheInterceptor.java

```diff
@@ -147,7 +145,14 @@ public void process(final HttpRequest request, final EntityDetails entity, final
         requestBuilder.headers(headerArrayToMap(request.getHeaders()));
 
         ExecutionAttributes attributes = new ExecutionAttributes();
-        attributes.putAttribute(AwsSignerExecutionAttribute.AWS_CREDENTIALS, awsCredentialsProvider.resolveCredentials());
+        // TODO: below line are for testing, remove later
+        AwsCredentials awsCredentials = null;
+        String accessKey = "accessKey";
+        String secretKey = "secretKey";
+        //awsCredentials = AwsBasicCredentials.create(accessKey, secretKey);
+
+        awsCredentials = awsCredentialsProvider.resolveCredentials();
```

Comment:
@mallikagogoi7 Is this ready for review? Looks like you still have testing code here to be cleaned up.

Reply:
@oeyh, This piece of code was pushed for testing purpose, we have resolve the issue and commited